src/lightning_classify/data/datamodule_v2.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+from pathlib import Path
+
+from torchvision.datasets import ImageFolder
+from torch.utils.data import DataLoader
+from lightning_classify.transforms import ImageTransform
+
+from pytorch_lightning import LightningDataModule
```

Comment:
from the way you import this, its not the newest version of pytorch lightning, find the minimum version that the way you use the import is ```from lightning import LightningDataModule```